BambooTracker/BambooTracker.pro

```diff
@@ -391,4 +390,38 @@ RESOURCES += \
 TRANSLATIONS += \
     res/lang/bamboo_tracker_fr.ts
 
+equals(QT_MAJOR_VERSION, 5) {
+    lessThan(QT_MINOR_VERSION, 12) {
+        message(using a workaround for missing 'lrelease' option in Qt <5.12...)
+        
+        for(tsfile, TRANSLATIONS) {
+          qmfile   = $$tsfile
+          qmfile  ~= s/.ts$/.qm/
+          qmlang   = $$tsfile
```

Comment:
The config `lrelease` would place translations under the directory `.qm`.
I would reciprocate this in the destination filename as well.

```suggestion
          qmfile  ~= s%^res/lang/%.qm/%
```

I would edit the generating command to match.
```
$thisqm = mkdir -p .qm && lrelease -qm $$qmfile $$tsfile
```

It's somewhat useful, because I designated this `.qm` folder as the place to search for translations in source tree, while in debug configuration.

Reply:
Hopefully added in https://github.com/rerrahkr/BambooTracker/pull/61/commits/ee82d87f261e20d6248ae21984704e22819cecd6.
I noticed a problem with the resulting Makefile install rules if the .qm file didn't already exist when qmake was run, so I had to add workaround system() calls to

1. test if the directory .qm/ exists. if not, create it.
2. test if the target .qm file already exists. if not, touch it with the modification date set to the epoch.

Let me know if there's something better I could've done, I'm not too deep into qmake and Qt yet.

Reply:
Please try this method instead.

```
            thisinstconf = $${thisinst}.CONFIG
          $$thisinstconf = no_check_exist
```

as discussed here: https://bugreports.qt.io/browse/QTBUG-47397

Reply:
Added, thank you! https://github.com/rerrahkr/BambooTracker/pull/61/commits/1f29f6a734499801b0231e16be5de0d0414364ae